tests/runner-tests/api-version/package.json

```diff
@@ -2,12 +2,12 @@
   "name": "api-version",
   "version": "0.1.0",
   "scripts": {
-    "codegen": "graph codegen --skip-migrations",
+    "codegen": "graph --version # dummy, we need a  'codegen'",
```

Comment:
Not sure what is happening here?

Reply:
The Rust framework in `tests/tests/runner_tests.rs` that is running these tests always runs `yarn codegen` so we need something there that succeeds; doing that was easier than modifying that framework